Signal/src/call/Speakerbox/CallKitCallUIAdaptee.swift

```diff
@@ -124,12 +126,6 @@ final class CallKitCallUIAdaptee: NSObject, CallUIAdaptee, CXProviderDelegate {
 
         disableUnsupportedFeatures(callUpdate: update)
 
-        // Update the provider configuration to reflect the caller's ringtone.
-        let sound = OWSSounds.ringtoneSound(for: call.thread)
-        let providerConfiguration = provider.configuration
-        providerConfiguration.ringtoneSound = OWSSounds.filename(for: sound)
```

Comment:
conflict heads up - this code has been removed in master already (https://github.com/signalapp/Signal-iOS/commit/c66fb70b2f2cf477adffa045e2a742bc1f46efcf)

Reply:
k